### PR TITLE
docs: documentation improvements in `search/linear_search.cpp`

### PR DESCRIPTION
**Pull request by ritikaa17**
Update readability of code

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1641"></a>

### CODE REVIEW

**Review by Qannaf — APPROVED** [review]

**Review by arunabh-cloud — COMMENTED** [review]
Hi,
May I suggest some improvements to this code? 
Thanks.

File: search/linear_search.cpp [hunk 1]
```diff
@@ -32,7 +35,7 @@ int main() {
     int *array = new int[size];
     int key;
 
-    // Input array
+    /*User gets to input array elements*/
     std::cout << "\nEnter the Array of " << size << " numbers : ";
```
Comment (arunabh-cloud) [inline search/linear_search.cpp:39]: There should be a check for the user entering a negative number or a very large number as "size" of the array of numbers.

**Review by Panquesito7 — DISMISSED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by mishraabhinn — CHANGES_REQUESTED** [review]
Can you provide the test cases also @ritikaa17. Thanks

**Review by mishraabhinn — APPROVED** [review]

**Pull request merged**